src/test/java/org/fundacionjala/coding/jorge/FizzBuzzTest.java

```diff
@@ -0,0 +1,82 @@
+package org.fundacionjala.coding.jorge;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+/**
+ *
+ */
+public class FizzBuzzTest {
+    /**
+     *
+     */
+    private FizzBuzz f = new FizzBuzz();
+
+    /**
+     *
+     */
+    @Test
+    public void shouldConvertThreeToFizz() {
+        final Object[] expected = new Object[]{2, "Fizz"};
+        final int[] actual = new int[]{2, 3};
+        assertArrayEquals(expected, f.fizzBuzz(actual));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void shouldConvert6And9ToFizz() {
+        final Object[] expected = new Object[]{"Fizz", 7, 8, "Fizz"};
+        final int[] actual = new int[]{6, 9};
+        assertArrayEquals(expected, f.fizzBuzz(actual));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void shouldConvert5ToBuzz() {
+        final Object[] expected = new Object[]{"Buzz"};
+        final int[] actual = new int[]{5, 5};
+        assertArrayEquals(expected, f.fizzBuzz(actual));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void shouldConvert15ToFizzBuzz() {
+        final Object[] expected = new Object[]{"FizzBuzz"};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "FizzBuzz" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)